Source/ASCollectionView.mm

```diff
@@ -916,7 +916,9 @@ - (CGSize)collectionView:(UICollectionView *)collectionView layout:(UICollection
   ASCellNode *cell = [self nodeForItemAtIndexPath:indexPath];
   if (cell.shouldUseUIKitCell) {
     if ([_asyncDelegate respondsToSelector:@selector(collectionView:layout:sizeForItemAtIndexPath:)]) {
-      return [(id)_asyncDelegate collectionView:collectionView layout:collectionViewLayout sizeForItemAtIndexPath:indexPath];
+      CGSize size = [(id)_asyncDelegate collectionView:collectionView layout:collectionViewLayout sizeForItemAtIndexPath:indexPath];
+      cell.style.preferredSize = size;
```

Comment:
Respect my feedback or not, but ASDK internally should never ever have to set a `preferredSize` to any layout element.